Scripts/Python/plasma/Plasma.py

```diff
@@ -1476,6 +1476,10 @@ def exitSubWorld(self):
         """Exits the avatar from the subWorld where it was"""
         pass
 
+    def findBone(self, bone_name):
+        """Find the ptSceneObject for the requested bone"""
+        pass
+
```

Comment:
I think this stub is already formatted correctly, but it would be good to rebase this PR and use generate_stubs.py to do this part automatically.